autotune/data_selection_window.py

```diff
@@ -85,13 +92,48 @@ def openFile(self):
             self.combo_y.clear()
             self.combo_y.addItems(list_names)
 
+            # Trigger preset selection
+            self.combo_preset.setCurrentIndex(0)
+            self.selectPreset(0)
+
     def printRangeError(self):
         msg = QMessageBox()
         msg.setIcon(QMessageBox.Critical)
         msg.setWindowTitle("Error")
         msg.setText("Range is invalid")
         msg.exec_()
 
+    def selectPreset(self, index):
```

Comment:
Do you want to add a VTOL preset as well, with it selecting the second instance of the torque setpoints? Or detect that the ulog is from a VTOL?

Reply:
yes, that's a good idea